src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -39,6 +39,12 @@ import org.gradle.util.GradleVersion
 @Suppress("UnstableApiUsage")
 class NexusPublishPlugin : Plugin<Project> {
 
+    companion object {
+        //visibility for testing
+        const val SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME = "closeAndReleaseStagingRepository"
+        const val SIMPLIFIED_LEGACY_CLOSE_AND_RELEASE_TASK_NAME = "closeAndReleaseRepository"
```

Comment:
How strongly do you feel about adding this legacy task? My gut feeling is that we don't need it.

Reply:
Probably you are right. People will need to migrate to the new task name sooner or later anyway, so why postpone something what is inevitable :).